test_rmw_implementation/test/test_subscription.cpp

```diff
@@ -150,6 +150,14 @@ TEST_F(CLASSNAME(TestSubscription, RMW_IMPLEMENTATION), create_with_bad_argument
   EXPECT_EQ(nullptr, sub);
   rmw_reset_error();
 
+  rosidl_message_type_support_t * non_implementation_ts = const_cast<rosidl_message_type_support_t *>(ts);
```

Comment:
@Lobotuerk nit: 
```suggestion
  rosidl_message_type_support_t * non_const_ts = const_cast<rosidl_message_type_support_t *>(ts);
```
Same elsewhere.